html/components/spinners.js

```diff
@@ -32,14 +32,20 @@ const setSpinning = (element, options) => {
   const spinningAriaLabel = options.ariaLabel || 'Loading';
   const ariaValueText = options.ariaValueText || 'Loading';
   const role = options.role || 'progressbar';
+  const doNotDisable = options.doNotDisable || false;
 
   el.classList.add('sprk-c-Button--has-spinner');
   el.setAttribute('aria-label', spinningAriaLabel);
   el.setAttribute('data-sprk-spinner-text', el.textContent);
-  el.setAttribute('disabled', '');
   el.setAttribute('data-sprk-has-spinner', 'true');
   el.setAttribute('style', `width: ${width}px`);
 
+  // This flag should be used for submit buttons so that
+  // the disabled attribute does not suppress the submit behavior.
+  if (!doNotDisable) {
```

Comment:
this double negative is throwing me off, perhaps something more brain friendly like `hasDoNotDisable`

Reply:
it's only *slightly* less confusing though so if you have a better idea, go for it haha